impeller/entity/geometry.h

```diff
@@ -35,112 +50,192 @@ class Geometry {
 
   static std::unique_ptr<Geometry> MakeVertices(Vertices vertices);
 
-  static std::unique_ptr<Geometry> MakePath(Path path);
+  static std::unique_ptr<Geometry> MakeFillPath(Path path);
+
+  static std::unique_ptr<Geometry> MakeStrokePath(
+      Path path,
+      Scalar stroke_width = 0.0,
+      Scalar miter_limit = 4.0,
+      Cap stroke_cap = Cap::kButt,
+      Join stroke_join = Join::kMiter);
 
   static std::unique_ptr<Geometry> MakeCover();
 
-  virtual GeometryResult GetPositionBuffer(
-      std::shared_ptr<Allocator> device_allocator,
-      HostBuffer& host_buffer,
-      std::shared_ptr<Tessellator> tessellator,
-      ISize render_target_size) = 0;
+  virtual GeometryResult GetPositionBuffer(const ContentContext& renderer,
+                                           const Entity& entity,
+                                           RenderPass& pass) = 0;
```

Comment:
Passing around the `ContentContext` & `Entity` SGTM. The filter graph started off with minimal context being passed around and quickly ended up in a similar place.